.github/workflows/ci.yml

```
@@ -43,68 +88,23 @@
-        with:
-          java-version: '21'
-          distribution: 'temurin'
-      - run: |
-          cd services/auth-service
-          docker build -t auth-service .
+        run: |
+          ./mvnw -B verify sonar:sonar \
+          -DskipTests=false \
+          -Dsonar.projectKey=AndriiSynytsia_subscription-tracker \
+          -Dsonar.junit.reportPaths=**/taget/surefire-reports \
+          -Dsonar.coverage.jacoco.xmlReportPaths=**/target/site/jacoco/jacoco.xml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)